<ui version="4.0" >
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_3" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="textBrowser" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>321</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3" >
           <widget class="QPushButton" name="textitalicButton" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../images.qrc" >:/images/edit-underline.png</iconset>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="textunderlineButton" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../images.qrc" >:/images/edit-italic.png</iconset>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="textboldButton" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="../images.qrc" >:/images/edit-bold.png</iconset>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QPushButton" name="colorButton" >
            <property name="minimumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item rowspan="2" row="0" column="1" >
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="avatarFrameButton" >
        <property name="minimumSize" >
         <size>
          <width>14</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>14</width>
          <height>32</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../images.qrc" >:/images/show_toolbox_frame.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>14</width>
          <height>31</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>false</bool>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>509</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item rowspan="2" row="0" column="0" >
       <widget class="QTextEdit" name="lineEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textboldButton</tabstop>
  <tabstop>textunderlineButton</tabstop>
  <tabstop>textitalicButton</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>avatarFrameButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
